modules/plugins/wfly-10/src/main/java/org/rhq/modules/plugins/wildfly10/ConnectorDiscoveryGroupValidatorComponent.java

```diff
@@ -172,8 +174,17 @@ public Operation getBatchOperation() {
                     .getSimpleValue(DISCOVERY_GROUP_NAME)));
             } else {
                 compositeOperation.addStep(new UndefineAttribute(address, DISCOVERY_GROUP_NAME));
-                compositeOperation.addStep(new WriteAttribute(address, CONNECTOR_ATTRIBUTE, Collections.singletonMap(
-                    configuration.getMap(CONNECTOR_PROPERTY).getSimpleValue("name:0", EMPTY_STRING), null)));
+                List<Property> propertyList = configuration.getList(CONNECTOR_PROPERTY).getList();
+                List<String> connectors = new ArrayList<String>(propertyList.size());
+                for (Property property : propertyList) {
+                    if (property instanceof PropertySimple) {
+                        PropertySimple propertySimple = (PropertySimple) property;
+                        connectors.add(propertySimple.getStringValue());
+                    } else {
+                        getLog().warn(property.getName() + " property has unexpected type: " + property.getClass());
```

Comment:
Is this warning logging of any use? Should you throw Exception or just ignore the thing? Or log in debug mode at most?